apps/hannk/halide/CMakeLists.txt

```diff
@@ -8,49 +8,49 @@ set(CMAKE_CXX_EXTENSIONS NO)
 
```

Comment:
I don't think we need lines 1-7 of this file anymore -- I added them because my approach required a 'standalone' CMake file, but I don't think yours does.

Reply:
That's correct. This doesn't need to be standalone anymore.